src/components/Exchange/FeesTooltip.tsx

```diff
@@ -22,8 +22,7 @@ function getFeeLabel(type: FeeType) {
     deposit: t`Deposit Fee`,
     execution: t`Execution Fee`,
   };
-
-  return i18n._(labels[type]);
+  return i18n._(/*i18n*/ labels[type]);
```

Comment:
mm, is /*i18n*/ some kind of magic comment for lingui? Or can it be deleted?

Btw, probably this syntax is not needed as getFeeLabel is called inside component during render so language should update correctly with just 

```
return labels[type]?
```

Reply:
Yeah, I tested it and works well. I have removed it now.